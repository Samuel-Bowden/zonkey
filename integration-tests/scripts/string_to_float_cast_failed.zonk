start {
	println(string_to_float("23548.2482345.2345234f"));
}
