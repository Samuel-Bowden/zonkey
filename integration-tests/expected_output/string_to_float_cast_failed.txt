(EXCEPTION) FailedStringToFloatCast
  Failed to convert the provided String value into a Float.
	2 | 	println(string_to_float(<"23548.2482345.2345234f">));
